uat/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCControlServer.java

```diff
@@ -131,26 +144,27 @@ public void createMqttConnection(MqttConnectRequest request,
 
             MqttProtoVersion version = request.getProtocolVersion();
             if (version != MqttProtoVersion.MQTT_PROTOCOL_V50) {
-                logger.atWarn().log("MQTT_PROTOCOL_V50 is only supported but {} requested", version);
+                logger.atWarn().log("invalid protocolVersion {}, {} is only supported",
+                                        version, MqttProtoVersion.MQTT_PROTOCOL_V50);
                 responseObserver.onError(Status.INVALID_ARGUMENT
-                                .withDescription("invalid protocolVersion, only MQTT_PROTOCOL_V50 is supported")
+                                .withDescription("invalid protocolVersion, only MQTT_PROTOCOL_V50 supported")
```

Comment:
we will need to be able to use mqtt 3 as well in the near future.

Reply:
Its is possible but was not evaluated for phases 1 and 2. I added in gRPC protocol that ability but not in that MQTT5 client. To implement MQTT3.11 client we need add another implementations of MqttConnection and extend MqttLibImpl due to not only another client is used but also all underlying objects like options, arguments, packets and so one.